src/cascadia/TerminalSettingsEditor/AddProfile.h

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "AddProfile.g.h"
+#include "AddProfilePageNavigationState.g.h"
+#include "Utils.h"
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    struct AddProfilePageNavigationState : AddProfilePageNavigationStateT<AddProfilePageNavigationState>
+    {
+    public:
+        AddProfilePageNavigationState(const Model::CascadiaSettings& settings) :
+            _Settings{ settings } {}
+
+        void RequestAddNew()
+        {
+            _AddNewHandlers(nullptr, nullptr);
+        }
+
+        void RequestDuplicate(GUID profile)
+        {
+            _AddNewHandlers(nullptr, winrt::box_value(profile));
+        }
+
+        GETSET_PROPERTY(Model::CascadiaSettings, Settings, nullptr)
+        TYPED_EVENT(AddNew, Windows::Foundation::IInspectable, Windows::Foundation::IInspectable);

```

Comment:
Hmm... Maybe you can use `WINRT_CALLBACK` here since you're not using the "sender"?